<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_scale_window</class>
 <widget class="QDialog" name="set_scale_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>327</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Set Scale</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="setScaleInputField_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string> Enter value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setScaleInputField_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string> Enter value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="setScaleInputField_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string> Enter value</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>151</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance in pixels:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Known distance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Pixel aspect rato:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_scale_result">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale: 71.984 pixels/unit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>set_scale_window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>set_scale_window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
